### PR TITLE
Complete Rework for Russian Localization

### PR DESCRIPTION
**Pull request by Crabanutij**
-Most of the translations become more clear and understandable.
-Done with beautifying text, phrases and file in general
What these sussy indents/spaces doing there?
-For easier reading and editing the file. No they dont change the mod or crash the game. I have tested.

### CODE REVIEW

**Comment by rbasamoyai** [comment]
Bit odd why there's a conflict
also, as for the indentation suggestion: I implemented that and will be pushing it with a few other things. I'm not exactly sure how to resolve this, so maybe try working on that; if not I'll try fixing it myself. And also remove the indents as mentioned above

**Comment by rbasamoyai** [comment]
Indents implemented in https://github.com/rbasamoyai/CreateBigCannons/commit/6f0faf741c6e43d3dbb678fc7bde6860b49ba760

**Comment by Crabanutij** [comment]
Looks like it was arguing about positions of the strings. i resolved it

File: src/main/resources/assets/createbigcannons/lang/ru_ru.json [hunk 1]
```diff
@@ -1,120 +1,147 @@
 {
+  "itemGroup.createbigcannons": "Create: Big Cannons",
   "block.createbigcannons.bag_of_grapeshot": "Мешок с дробью",
+  "block.createbigcannons.he_shell": "Фугасный снаряд",
+  "block.createbigcannons.shrapnel_shell": "Шрапнельный снаряд",
+  "block.createbigcannons.solid_shot": "Тяжёлый снаряд",
+  
   "block.createbigcannons.bag_of_grapeshot.tooltip": "МЕШОК С ДРОБЬЮ",
-  "block.createbigcannons.bag_of_grapeshot.tooltip.behaviour1": "Выпускает крупную дробь. _Разделяется на части после выстрела._",
+  "block.createbigcannons.bag_of_grapeshot.tooltip.behaviour1": " Выпускает крупную дробь. _Разделяется_ на _части_ после выстрела.",
```
Comment (rbasamoyai) [inline src/main/resources/assets/createbigcannons/lang/ru_ru.json]: Indent

**Comment by rbasamoyai** [comment]
I mean, the squib ratio doesn't have to be barrels per 1 charge - in fact, bronze has a ratio of 3 : 2

**Comment by Crabanutij** [comment]
It's confusing me. Squib ratio should be like normal ratio instead of thi

s

**Comment by rbasamoyai** [comment]
Again, cannon info without goggles is low-info; as well, Create's bar making method is a bit limited

**Comment by rbasamoyai** [comment]
Looks good

**Comment by rbasamoyai** [comment]
hold up, I haven't pulled it yet

**Comment by rbasamoyai** [comment]
there we go

**Pull request merged**